src/App.jsx

```diff
@@ -1,28 +1,48 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
-import { TodoList } from './components/TodoList';
-import { CurrentUser } from './components/CurrentUser';
+import CurrentUser from './components/CurrentUser/CurrentUser';
+import TodoList from './components/TodoList/TodoList';
+import { getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
-    selectedUserId: 0,
+    selectedUserId: null,
   };
 
+  async componentDidMount() {
+    this.setState({
+      todos: await getTodos(),
+    });
```

Comment:
как ни странно для меня нагляднее так
```suggestion
    const todos = await getTodos();

    this.setState({ todos });
```